gstd/gstd_bus_msg.c

```diff
@@ -28,6 +28,7 @@
 #include "gstd_bus_msg_simple.h"
 #include "gstd_bus_msg_state_changed.h"
 #include "gstd_bus_msg_stream_status.h"
+#include "gstd_json_builder.h"
```

Comment:
I don't like coupling the individual objects to a specific format. Moreover it is indirect code replication, if we wanted to change to `xml` (hopefully we never go this mad) we'd need to change every object.

Reply:
What if we saved the `GType` in the object instance struct, functioning as a "factory".  So later on we use 
```c
GstdIFormatter *formatter = g_object_new (seff->formatter_factory, NULL);`
```